src/app/admin/_components/faq/hooks/useActiveFaqs.js

```diff
@@ -0,0 +1,18 @@
+import { useGetList } from 'react-admin';
+
+export const useActiveFaqs = ({ page, perPage } = { page: 1, perPage: 50 }) => {
+  const {
+    data: faqs = [],
+    isLoading,
+    error,
+  } = useGetList('faq', {
+    pagination: { page, perPage },
+  });
+  if (isLoading || error) {
+    return { total: 0, data: [], isLoading, error };
+  }
+
+  const sorted = faqs.map(({ id, isActive }) => ({ id, isActive })).sort(a => a.isActive);
```

Comment:
It is better to filter first in terms of performance. You can chain all three iteration and put the result into a single variable. Eg: `const mappedFaqs = faqs.filter().sort().map()`

Reply:
Thanks for comment! Fixed this.